tensorboard/data/server/data_compat.rs

```diff
@@ -286,6 +287,50 @@ impl SummaryValue {
     }
 }
 
+fn tf1x_image_metadata() -> Box<pb::SummaryMetadata> {
+    let plugin_content = pb::ImagePluginData {
+        converted_to_tensor: true,
+        ..Default::default()
+    };
+    let mut encoded_content = Vec::new();
+    plugin_content
+        .encode(&mut encoded_content)
+        // vectors are resizable, so should always be able to encode
+        .expect("failed to encode image metadata");
+
+    Box::new(pb::SummaryMetadata {
```

Comment:
Optional, but might be nice to have `blank_metadata_with_plugin_content()` to factor out the shared bit of these two helpers and `blank()`?  I guess I could go either way; up to you.

Reply:
Sure; no strong preference. Works out to net ±0 LOC. :-)